python/src/worker/tasks/__init__.py

```diff
@@ -14,3 +14,6 @@ def compute(self):
     @abstractmethod
     def _format_result(self, result):
         ...
+
+    def set_error(self, response):
+        self.error = response.get("error", False)
```

Comment:
this should not be handled here, see comment below

Reply:
removed